assets/startup.sh

```diff
@@ -1,8 +1,47 @@
 #!/bin/sh
+set -eu
+
+## Script will be run as chrony user !
 
 DEFAULT_NTP="0.pool.ntp.org,1.pool.ntp.org,2.pool.ntp.org,3.pool.ntp.org"
 CHRONY_CONF_FILE="/etc/chrony/chrony.conf"
 
+echo "
+-------------------------------------
+ℹ️ Container Information
+-------------------------------------
+OS:            $(. /etc/os-release; echo "${PRETTY_NAME}")
+Docker user:   $(whoami)
+Docker uid:    $(id -u)
+Docker gid:    $(id -g)
+"
+
+CURRENT_USERGROUP="$(id -u):$(id -g)"
+CHRONY_USERGROUP="$(id -u chrony):$(id -g chrony)"
+
+if [ "$CURRENT_USERGROUP" != "$CHRONY_USERGROUP" ]; then
+  echo "[WARNING] Container running under a different user than recommended!"
+  echo ""
+fi
+
+# Check if existing volumes/folders are writable
+if [ ! -w "/etc/chrony" ] || [ ! -w "/run/chrony" ] || [ ! -w "/var/lib/chrony" ]; then
```

Comment:
Aha, the sticky-bit would allow another user than chrony to write here. (my comment above on the Dockerfile change)